eth_utils/string.py

```diff
@@ -42,8 +159,11 @@ def force_obj_to_bytes(obj):
 
 
 def force_obj_to_text(obj):
```

Comment:
didn't deprecate this because it now uses `to_text` instead of `force_text`. would changing the name of this (& `force_obj_to_bytes` -> something like `convert_obj_*` be better / necessary?